### PR TITLE
Use cli_pkg for releases

### PR DESCRIPTION
**Pull request by jathak**

### CODE REVIEW

**Comment by jathak** [comment]
@nex3: Can you add `GITHUB_TOKEN` and `NPM_TOKEN` to the Travis config?

File: test/utils.dart [hunk 1]
```diff
@@ -6,50 +6,36 @@
...
 import 'dart:io';
 
+import 'package:cli_pkg/testing.dart' as pkg;
 import 'package:path/path.dart' as p;
 import 'package:test/test.dart';
 import 'package:test_descriptor/test_descriptor.dart' as d;
 import 'package:test_process/test_process.dart';
 
+const jsExecutable = "build/sass-migrator.dart.js";
+const dartSnapshot = "build/sass_migrator.dart.snapshot";
```
Comment (nex3) [inline test/utils.dart:16]: What are these used for?

File: tool/grind.dart [hunk 2]
```diff
@@ -2,13 +2,31 @@
...
+
 export 'grind/sanity_check.dart';
-export 'grind/standalone.dart';
 
-main(List<String> args) => grind(args);
+main(List<String> args) {
+  pkg.humanName = "Sass Migrator";
+  pkg.botName = "Sass Bot";
+  pkg.botEmail = "sass.bot.beep.boop@gmail.com";
+  pkg.executables = {"sass-migrator": "bin/sass_migrator.dart"};
```
Comment (nex3) [inline tool/grind.dart:18]: This is already the autodetected value.

File: tool/grind.dart [hunk 3]
```diff
@@ -2,13 +2,31 @@
...
+  pkg.homebrewRepo = "sass/homebrew-sass";
+  pkg.homebrewFormula = "migrator.rb";
+  pkg.jsRequires = {"fs": "fs", "os": "os", "path": "path"};
+  pkg.npmPackageJson =
+      json.decode(File("package/package.json").readAsStringSync())
+          as Map<String, Object>;
```
Comment (nex3) [inline tool/grind.dart:24]: Might it make sense to just move `package.json` to the root so this is autoloaded?

File: tool/grind.dart [hunk 4]
```diff
@@ -2,13 +2,31 @@
...
+  pkg.npmReadme = File("README.md").readAsStringSync();
```
Comment (nex3) [inline tool/grind.dart:25]: This is already the autodetected value.

File: tool/grind.dart [hunk 5]
```diff
@@ -2,13 +2,31 @@
...
+  pkg.standaloneName = "sass-migrator";
+
+  pkg.githubReleaseNotes = "";
```
Comment (nex3) [inline tool/grind.dart:28]: What's up with this?

File: tool/grind/utils.dart [hunk 6]
```diff
@@ -3,43 +3,25 @@
...
-
-/// The version of the migrator.
-final String version =
-    loadYaml(File('pubspec.yaml').readAsStringSync())['version'] as String;
-
-/// The version of the current Dart executable.
-final Version dartVersion = Version.parse(Platform.version.split(" ").first);
 
 /// Options for [run] that tell Git to commit using SassBot's name and email.
 final sassBotEnvironment = RunOptions(environment: {
```
Comment (nex3) [inline tool/grind/utils.dart:13]: Is this actually still used anywhere? Same question for a bunch of the other members in here.

File: .travis.yml [hunk 7]
```diff
@@ -29,14 +29,12 @@ jobs:
 
   # Testing on Node.js
   - env: NODE_VERSION=stable
-    # Remove once pulyaevskiy/node-interop#74 is fixed.
-    dart: &node-compatible-dart
-      2.7.2
     dart_task: {test: -t node}
   # Keep these up-to-date with the latest LTS Node releases. They next need to
-  # be rotated April 2020. See https://github.com/nodejs/Release.
```
[the remark this answers is not in the document]
Comment (jathak) [reply .travis.yml:37]: Will do

File: test/utils.dart (hunk 1 above)
Comment (jathak) [reply test/utils.dart]: Whoops. These were leftover from before I fixed the bug in `cli_pkg`. Removed.

File: tool/grind.dart (hunk 2 above)
Comment (jathak) [reply tool/grind.dart]: Removed

File: tool/grind.dart (hunk 3 above)
Comment (jathak) [reply tool/grind.dart]: Done

File: tool/grind.dart (hunk 4 above)
Comment (jathak) [reply tool/grind.dart]: Removed

File: tool/grind.dart (hunk 5 above)
Comment (jathak) [reply tool/grind.dart]: Removed

File: tool/grind/utils.dart (hunk 6 above)
Comment (jathak) [reply tool/grind/utils.dart]: I originally just modified this to remove the same members that Dart Sass did, but, yeah, it looks like the only one that was actually still used was `environment`. I just moved it and the sanity check that used it into `tool/grind.dart` and deleted the `tool/grind` directory entirely.

**Pull request merged**